addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -629,6 +630,24 @@ export class OdooEditor extends EventTarget {
                 }
 
                 attributeCache.set(record.target, attributeCache.get(record.target) || {});
+                if (record.attributeName === 'class') {
+                    const classBefore = record.oldValue?.split(' ') || [];
+                    const classAfter = record.target.className.split(' ');
+                    const excluded = [];
+                    for (const klass of classBefore) {
```

Comment:
Why name the variable `klass` ? There doesn't seem to be another variable named `class` that would require the name change.

Reply:
Because `class` is a reserved keyword.

Reply:
Right 🤦